pkgs/development/python-modules/lektor/default.nix

```diff
@@ -12,35 +12,47 @@
 , flask
 , pyopenssl
 , ndg-httpsclient
-, pkgs
+, pytest
+, pytestcov
+, pytest-mock
+, pytest-pylint
+, pytest-click
+, isPy27
+, functools32
 }:
 
 buildPythonPackage rec {
   pname = "lektor";
-  version = "2.3";
+  version = "3.1.3";
 
-  src = fetchgit {
-    url = "https://github.com/lektor/lektor";
-    rev = "refs/tags/${version}";
-    sha256 = "1n0ylh1sbpvi9li3g6a7j7m28njfibn10y6s2gayjxwm6fpphqxy";
+  src = fetchFromGitHub {
+    owner = "lektor";
+    repo = "lektor";
+    rev = version;
+    sha256 = "16qw68rz5q77w84lwyhjpfd3bm4mfrhcjrnxwwnz3vmi610h68hx";
   };
 
-  buildInputs = [ pkgs.glibcLocales ];
   propagatedBuildInputs = [
     click watchdog exifread requests mistune inifile Babel jinja2
     flask pyopenssl ndg-httpsclient
+  ] ++ lib.optionals isPy27 [ functools32 ];
+
+  checkInputs = [
+    pytest pytestcov pytest-mock pytest-pylint pytest-click
   ];
 
-  LC_ALL="en_US.UTF-8";
+  checkPhase = ''
```

Comment:
Is this really necessary given that there is a `doCheck = false` lower down?

I think it could be removed since it is unused.

Reply:
Yes it could. I've always debated with myself on this. I have the reasoning in comments "# many errors -- tests assume inside of git repo, linting errors 13/317 fail". I am thinking it will make others work easier when the tests don't fail

Reply:
I recommend to keep this as a starting point for future efforts to reenable the tests.

Reply:
With that this should be resolved and be ready for merge.